WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/GeneralActivityListener.java

```diff
@@ -34,6 +38,9 @@ public void eventDispatched(AWTEvent event) {
         keyboardActivityListener = new AWTEventListener() {
             @Override
             public void eventDispatched(AWTEvent event) {
+                if(!WatchDogUtils.getProjectName().equals(projectName)) {
+                    return;
+                }
                 if (((KeyEvent) event).getKeyCode() == KeyEvent.VK_RIGHT ||
```

Comment:
Can you factor out the first conversion to make the if more readable? Moreover, why not use a switch-case statement?